unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackPreviewPanel.cs

```diff
@@ -38,24 +59,51 @@ public void Initialize(ICharacterPreviewFactory characterPreviewFactory)
                 loadingMode: CharacterPreviewMode.WithoutHologram,
                 renderTexture: (RenderTexture) avatarPreviewImage.texture,
                 isVisible: false,
-                cameraFocus: CharacterPreviewController.CameraFocus.DefaultEditing,
+                previewCameraFocus: PreviewCameraFocus.DefaultEditing,
                 isAvatarShadowActive: true);
             characterPreviewController.SetCameraLimits(CAMERA_LIMITS);
             characterPreviewController.ConfigureZoom(
                 CAMERA_ZOOM_CENTER, CAMERA_ZOOM_BOTTOM_MAX_OFFSET, CAMERA_ZOOM_TOP_MAX_OFFSET);
-            characterPreviewController.SetFocus(CharacterPreviewController.CameraFocus.DefaultEditing);
-            avatarPreviewRotation.OnHorizontalRotation += OnPreviewRotation;
-            avatarPreviewPanning.OnPanning += OnPreviewPanning;
-            avatarPreviewZoom.OnZoom += OnPreviewZoom;
+            characterPreviewController.SetFocus(PreviewCameraFocus.DefaultEditing);
+
+            avatarPreviewRotationController = new PreviewCameraRotationController(
```

Comment:
You should be injecting this, otherwise it will be impossible to mock them.

Reply:
Done:
- https://github.com/decentraland/unity-renderer/pull/5162/commits/417944446fb76631500125cf1fb6c639814bc60c
- https://github.com/decentraland/unity-renderer/pull/5162/commits/eb99409140036c34e538a551ea7a1b15979d7ac0